src/AuditBundle.php

```diff
@@ -140,4 +139,21 @@ private function validate(array $config): void
             throw new InvalidConfigurationException('WhiteDigital\Audit: "audit_entity_manager" and "default_entity_manager" names must be different');
         }
     }
+
+    private function configureApiPlatformExtension(array $extensionConfig, ContainerBuilder $builder): void
```

Comment:
why copy logic from existing function?  
just replace self::PATHS in `$this->addApiPlatformPaths($container, self::PATHS);` with needed one: `[self::API_RESOURCE_PATH]` or `[$extensionConfig['custom_api_resource_path']]`

Reply:
Is logic the same?
The trait configures the extension via `ContainerConfigurator`
```
$container->extension('api_platform', [
    'mapping' => [
        'paths' => $paths,
    ],
]);
```

This function modifies the configuration for extension via `ContainerBuilder` as per documentation 
```
$builder->prependExtensionConfig('api_platform', [
    'mapping' => [
        'paths' => [self::API_RESOURCE_PATH],
    ],
]);
```
you tell me what are the differences, and what is the more effective way to modify extensions configuration

Reply:
only difference here is where you want your configuration to appear - after bundle's initial configuration (current version) or before bundle's initial configuration. both versions happen in prepend extension so this is still about just config:  
```php
[
    'paths' => [
        <api_platform_path>,
        <new_package_path>
    ]
]
```
or
```php
[
    'paths' => [
        <new_package_path>,
        <api_platform_path>
    ]
]
```
in my opinion appending paths would be more correct